### PR TITLE
caption added

### PR DESCRIPTION
**Pull request by murraygsutter**
Related to #2

### CODE REVIEW

File: _posts/0000-01-19-murraygsutter.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ title: "Welcome to our slide deck!"
 ---
 
 ![success-kid](https://cloud.githubusercontent.com/assets/16547949/25401204/136e768c-29c3-11e7-8a7b-8724fee13dd7.jpg)
-{: .center}
+{: .center
 
-Caption here
+Mine!
```
Comment (murraygsutter) [inline _posts/0000-01-19-murraygsutter.md]: ha ha

Comment (bailyd) [inline _posts/0000-01-19-murraygsutter.md]: Looks good.

Comment (murraygsutter) [inline _posts/0000-01-19-murraygsutter.md]: funny

File: _posts/0000-01-19-murraygsutter.md [hunk 2]
```diff
@@ -4,6 +4,6 @@ title: "Welcome to our slide deck!"
 ---
 
 ![success-kid](https://cloud.githubusercontent.com/assets/16547949/25401204/136e768c-29c3-11e7-8a7b-8724fee13dd7.jpg)
-{: .center}
+{: .center
```
Comment (bailyd) [inline _posts/0000-01-19-murraygsutter.md]: I wonder what this mark up is for?

**Pull request merged**